integration-cli/docker_api_network_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/docker/docker/api"
+	"github.com/go-check/check"
+)
+
+func isNetworkAvailable(c *check.C, name string) bool {
+	status, body, err := sockRequest("GET", fmt.Sprintf("/v%s/networks", api.APIVERSION), nil)
```

Comment:
Correct. that is because of https://github.com/docker/libnetwork/pull/209. once this PR is resolved in libnetwork and vendored in, we don't need the version parameter.
But, it is not wrong to have the version parameter (since Docker supports both the base-urls and hence both must work). This being just the Test code, we should be okay. But we will vendor in the above fix soon.